docs/source/markdown/podman-healthcheck.1.md

```diff
@@ -7,7 +7,37 @@ podman\-healthcheck - Manage healthchecks for containers
 **podman healthcheck** *subcommand*
 
 ## DESCRIPTION
-podman healthcheck is a set of subcommands that manage container healthchecks
+podman healthcheck is a set of subcommands that manage container healthchecks.
```

Comment:
`The **podman healthcheck** suite of commands are used to manage container healthchecks.`